latest/CGI/keywords/cgi_common.yaml

```
@@ -10,43 +10,93 @@ type: object
 properties:
   INSTRUME:
     title: Instrument
-    description: Placeholder
+    description: Instrument used
     type: string
     maxLength: 8
     archive_catalog:
       datatype: nvarchar(8)
-      destination: [CGIPlaceholder.placeholder]
+      destination: [CGIPlaceholder.instrument_name]
   PROGNUM:
     title: Program number
-    description: Placeholder
+    description: ID number (integer) assigned to the proposal
     type: string
     maxLength: 16
     archive_catalog:
       datatype: nvarchar(16)
-      destination: [CGIPlaceholder.placeholder]
+      destination: [CGIPlaceholder.program]
   SEGMENT:
     title: Segment number
-    description: Placeholder
+    description: The sequence of activities produced by each iteration of a Segment Plan (Observation Sequence) in a Pass
     type: string
     maxLength: 16
     archive_catalog:
       datatype: nvarchar(16)
-      destination: [CGIPlaceholder.placeholder]
+      destination: [CGIPlaceholder.segment]
   VISNUM:
     title: Visit number
-    description: Placeholder
+    description: Visit number within a program
     type: string
     maxLength: 16
     archive_catalog:
       datatype: nvarchar(16)
-      destination: [CGIPlaceholder.placeholder]
+      destination: [CGIPlaceholder.visit]
   DATALVL:
-    title: Data level
-    description: Placeholder
+    title: Product Levels
+    description: Product levels, 1 is raw uncalibrated images, 2a is calibrated images, 2b is coadded calibrated images, 3 is photometric-, astrometric-, and wavelength-calibrated images, 4 is PSF-subtracted and/or spectrally extracted data
     type: string
     maxLength: 16
     archive_catalog:
       datatype: nvarchar(16)
-      destination: [CGIPlaceholder.placeholder]
+      destination: [CGIPlaceholder.productLevel]
```

```suggestion
      destination: [CGIPlaceholder.product_level]
```
According to the agreed upon format for database destinations we expect these to follow this regex: https://github.com/spacetelescope/rad/blob/bd51157998b0507c23f79b0d9df94748a5ec6959/latest/rad_schema.yaml#L87. Which essentially says that the `bar` section of `Foo.bar` should be in snake case.